adding to new market proposal documentation (#103)
dydxprotocol/v4-documentation: pages/governance/proposing_a_new_market.md

- Reference price (e.g. `40,000`) and `p` value
  - Defines the exponent on the reference price as `p:=FLOOR(log10(reference_price))`. For example: `p:=FLOOR(log10(40,000))=4`
- Exchanges required for oracles (`exchange_config_json`)
- Liquidity Tier (can be updated through governance vote at a later date.)
  - 0: Large Cap (BTC/ETH)

>   - 1: Mid-Cap (Markets with at least 8 robust oracle sources with liquidity >= 50K on both sides and 30d daily spot trading volume >= $100M.)

Author: Think it depends on how conservative the recommendation for new markets should be. There are some markets with 7 oracles (APE and SEI) that are tier 2 because they are more recently launched. I think it might make sense for new markets to be recommended at long-tail unless considered safe.

| `Msg[Update/Create]ClobPair` | `step_base_quantums` | (aka step size): min increment in the size of orders (number of coins) on the CLOB in base quantums. | `1000000` |
| `MsgDelayMessage` | `delay_blocks` | number of blocks before which the `MsgUpdateClobPair` is executed and transitions the market to `ACTIVE` | `3600` (equal to an hour at `1 sec` blocktime) |

## Choosing oracle sources

To select oracle sources, follow the procedure below: 

1. Find spot market tickers from eligible exchanges with the desired symbol as the base asset and USD or USDT as the quote asset. See below for a list of eligible exchanges.
  - If the quote asset is USDT, add the following flag `"adjustByMarket":"USDT-USD"`. This flag ensures that the base asset oracle price is adjusted by the USDT oracle price. 
2. Currently the software supports only one ticker from an exchange per market. Among the tickers from an exchange, choose the most liquid market with the highest trading volume. If one market is more liquid but another has more trading volume, choose the one with deeper liquidity. 

> 3. Exclude markets that do not meet the depth and daily trading volume threshold over the past month. 

Reviewer: Should it be `Exclude oracle sources` instead of `Exclude markets` ?

  - Both sides of liquidity at 2% from the midprice should be at least `$50,000`. 
  - The average daily trading volume should be at least `$100,000`. 
4. Ensure there are at least 6 sources. If there are less than 6 sources, this market should not be added to prevent potential market manipulation attacks and consensus failures from oracle price updates unless more robust oracle sources are available.

### List of eligible exchanges

Recommended Exchanges:
- Binance
- Coinbase
- OKX
- Bybit
- Gate
- Kraken
- Kucoin

> - MEXC

Reviewer: Why did u collapse Tier 1/2 exchanges into one? Maybe we should add HTX/BItstamp in a new tier called "Only use if necessary" and have Tier 2 as `Use at discretion` ?

Author: Since we're specifying min 6 sources, wasn't sure if tier 1/2 distinction was still relevant since tier 2 exchanges would have to be used to meet this requirement. Updating HTX/Bitstamp to be only use if necessary.

| `Msg[Update/Create]ClobPair` | `step_base_quantums` | (aka step size): min increment in the size of orders (number of coins) on the CLOB in base quantums. | `1000000` |
| `MsgDelayMessage` | `delay_blocks` | number of blocks before which the `MsgUpdateClobPair` is executed and transitions the market to `ACTIVE` | `3600` (equal to an hour at `1 sec` blocktime) |

### Determining liquidity tier

> Liquidity tier is used to determine various risk and market parameters. A new market should be 2: Long-tail unless there are strong reasons to think that it is safe, in which case it can 1: Mid-cap. Liquidity tier and associated parameters can be updated through governance vote at a later date. A market may be considered for 1: Mid-cap if there are at least 8 robust spot markets with reasonable liquidity and 30d daily trading volume is at least $100M across exchanges.

Reviewer 2: Can we merge this section to above around line 38, to avoid scattered information?

Author: sounds good! qq: what does `so mappings could change if these tiers change` mean? Is it that the formula from liquidity tier to parameters can change later?

> ## Choosing oracle sources

Reviewer 2: Can you consolidate `Choosing oracle sources`, `List of eligible exchanges`, and `exchange_config_json` below into one section, and link to this section on line 50 where it says `See Below`?

Author: hmm happy to consolidate, but `list of eligible exchanges` and `exchange_config_json` are subsections under `Choosing oracle sources`. Do you think it's necessary to further consolidate?

  - If the quote asset is USDT, add the following flag `"adjustByMarket":"USDT-USD"`. This flag ensures that the base asset oracle price is adjusted by the USDT oracle price. 
2. Currently the software supports only one ticker from an exchange per market. Among the tickers from an exchange, choose the most liquid market with the highest trading volume. If one market is more liquid but another has more trading volume, choose the one with deeper liquidity. 
3. Exclude markets that do not meet the depth and daily trading volume threshold over the past month. 
  - Both sides of liquidity at 2% from the midprice should be at least `$50,000`. 
  - The average daily trading volume should be at least `$100,000`. 
4. Ensure there are at least 6 sources. If there are less than 6 sources, this market should not be added to prevent potential market manipulation attacks and consensus failures from oracle price updates unless more robust oracle sources are available.

### List of eligible exchanges

> 

Reviewer 2: ```suggestion
Recommended Exchanges:
```

- Binance
- Coinbase
- OKX
- Bybit
- Gate
- Kraken
- Kucoin
- MEXC

> (only consider on a discretionary basis)  

Reviewer 2: ```suggestion
Only consider below on a Discretionary Basic:
```

- HTX (previously Huobi)
- Bitstamp

> Exchanges not included in the above list are not currently supported by the software. Furthermore, we do not recommend tickers from the following exchanges as oracle sources:

Reviewer 2: Mega nit:
```suggestion
Exchanges not included in the above list are not currently supported by the software. Furthermore, the following exchanges are not recommended as oracle sources:
```
